helpers/cacheBuild.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+
+const DEFAULT_DIST_DIR = '.next'
+
+// Give advanced/monorepo users the ability to point to their custom build & dist dirs
```

Comment:
Instead of using a `buildDir` input, couldn't users use our `base` configuration property instead?
This changes the current directory of the plugin's process, which is used by default to resolve the file paths passed to `utils.cache.save()` and `utils.cache.restore()`.

Reply:
hm i suppose so, though i know *as is* (with current monorepo support) the base configuration doesn't work for all monorepo use cases - which i guess begs the qestion, is that why pizzafox's plugin accepts buildDir as input? 🤔

Reply:
The `base` property is definitely more of a patch than a real solution for many monorepo setups.

In this case though, I think the `buildDir` input would use the same strategy as the `base` property. In essence, it would behave the same by changing the base directory to find `.next`.

However, please feel free to discard this comment if you think this might be a useful input. :+1: 

If we do though, I would personally suggest renaming it to `base` because `buildDir` is a somewhat internal name, and to reflect the similar configuration property with the same name.

Reply:
ok no i can get rid of buildDir (or `base`) so we won't have any inputs after all! if people end up needing it, we can always add it back in :)